### DISCUSSION TITLE
How to make x-text not put font tags

### DISCUSSION THREAD

**Discussion by FuriosoJack** [alpinejs/alpine#1295] [1. Help]
I have been reviewing the rendered code when an x-text is put and what it does is put the text inside <fonts> tags which is causing me problems when translating the text and applying the styles

![image](https://user-images.githubusercontent.com/26976637/117305654-95d97e00-ae44-11eb-8ce1-91c5c3d0a6b9.png)

**Comment by KevinBatdorf** [comment]
You probably have a typo in your code. Can you share your code or put an example on codepen?
